clients/fides-js/src/components/tcf/TcfPurposes.tsx

```diff
@@ -174,14 +178,17 @@ const TcfPurposes = ({
               ]
             : activeData.enabledPurposeIds
         }
-        onToggle={(newEnabledIds, item) =>
-          onChange({
-            newEnabledIds,
-            // @ts-ignore
-            modelType: item.bestTranslation
-              ? "customPurposesConsent"
-              : activeData.purposeModelType,
-          })
+        onToggle={(newEnabledIds, item, toggleDetails) =>
+          onChange(
+            {
+              newEnabledIds,
+              // @ts-ignore
+              modelType: item.bestTranslation
```

Comment:
I know this isn't your doing, but it would be nice to make this a bit safer while we're here
```suggestion
              modelType: (item as PrivacyNoticeWithBestTranslation).bestTranslation
```

Reply:
Isn't that a 6 of one, half a dozen of another sort of change?

Reply:
Sort of. If there were other TS issues besides this one, they would not get seen. This is easier for future devs, basically.

Reply:
An example of where this could be problematic is if `bestTranslation` were also removed from `PrivacyNoticeWithBestTranslation` and that wouldn't get picked up here because of the `//@ts-ignore`

Reply:
Ah ok.